### PR TITLE
Runner installation updates

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
Update instructions for macOS and Linux runner installation.
Update the path for /opt/circleci

# Reasons
New/missing instructions for macOS and Linux install process.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Comment by sebastian-lerner** [comment]
@circleci/runner can someone take a quick peek and make sure this looks good?

File: jekyll/_cci2/runner-config-reference.adoc [hunk 1]
```diff
@@ -87,7 +87,7 @@ The custom script must:
 1. Take great care to ensure the task agent (that is, the arguments passed to it) runs
 2. Forward the exit code from task agent as its own exit code
 
-Example script saved as `/opt/circleci/wrapper.sh`:
+Example script saved as `var/opt/circleci/wrapper.sh`:
```
Comment (catherinepope) [inline jekyll/_cci2/runner-config-reference.adoc:90]:
```suggestion
Example script saved as `/var/opt/circleci/wrapper.sh`:
```

File: jekyll/_cci2/runner-config-reference.adoc [hunk 2]
```diff
@@ -104,7 +104,7 @@ Config snippet for above example:
 
 ```yaml
 runner:
-  command_prefix: ["/opt/circleci/wrapper.sh"]
+  command_prefix: ["var/opt/circleci/wrapper.sh"]
```
Comment (catherinepope) [inline jekyll/_cci2/runner-config-reference.adoc:107]:
```suggestion
  command_prefix: ["/var/opt/circleci/wrapper.sh"]
```

File: jekyll/_cci2/runner-config-reference.adoc [hunk 3]
```diff
@@ -119,7 +119,7 @@ Example:
 
 ```yaml
 runner:
-  working_directory: /opt/circleci/workdir
+  working_directory: var/opt/circleci/workdir
```
Comment (catherinepope) [inline jekyll/_cci2/runner-config-reference.adoc:122]:
```suggestion
  working_directory: /var/opt/circleci/workdir
```

File: jekyll/_cci2/runner-config-reference.adoc [hunk 4]
```diff
@@ -131,7 +131,7 @@ Example:
 
 ```yaml
 runner:
-  working_directory: /opt/circleci/%s
+  working_directory: var/opt/circleci/%s
```
Comment (catherinepope) [inline jekyll/_cci2/runner-config-reference.adoc:134]:
```suggestion
  working_directory: /var/opt/circleci/%s
```

File: jekyll/_cci2/runner-faqs.adoc [hunk 5]
```diff
@@ -116,8 +116,8 @@ Note that you must check to see if there are greater than two processes because
 Try running the following two commands:
 
 ```bash
-sudo chmod +x /opt/circleci/circleci-launch-agent
-sudo /opt/circleci/circleci-launch-agent --config=/Library/Preferences/com.circleci.runner/launch-agent-config.yaml
+sudo chmod +x var/opt/circleci/circleci-launch-agent
```
Comment (catherinepope) [inline jekyll/_cci2/runner-faqs.adoc:119]:
```suggestion
sudo chmod +x /var/opt/circleci/circleci-launch-agent
```

Comment (catherinepope) [inline jekyll/_cci2/runner-faqs.adoc:119]:
```suggestion
sudo chmod +x /var/opt/circleci/circleci-launch-agent
```

File: jekyll/_cci2/runner-faqs.adoc [hunk 6]
```diff
@@ -116,8 +116,8 @@ Note that you must check to see if there are greater than two processes because
...
+sudo var/opt/circleci/circleci-launch-agent --config=/Library/Preferences/com.circleci.runner/launch-agent-config.yaml
```
Comment (catherinepope) [inline jekyll/_cci2/runner-faqs.adoc:120]:
```suggestion
sudo /var/opt/circleci/circleci-launch-agent --config=/Library/Preferences/com.circleci.runner/launch-agent-config.yaml
```

File: jekyll/_cci2/runner-on-kubernetes.adoc [hunk 7]
```diff
@@ -60,7 +60,7 @@ $ kubectl logs -f circleci-runner-test-7d6b8fc6f-4z5wl
 
 ... (output truncated)
 
-time="2021-03-25T20:55:40Z" level=info msg="CircleCI launcher starting" config=/opt/circleci/launch-agent-config.yaml
+time="2021-03-25T20:55:40Z" level=info msg="CircleCI launcher starting" config=var/opt/circleci/launch-agent-config.yaml
```
Comment (catherinepope) [inline jekyll/_cci2/runner-on-kubernetes.adoc:63]:
```suggestion
time="2021-03-25T20:55:40Z" level=info msg="CircleCI launcher starting" config=/var/opt/circleci/launch-agent-config.yaml
```

**Review by catherinepope — COMMENTED** [review]
You might want to double-check with an engineer, but I think all the paths need a slash at the front.

**Comment by sebastian-lerner** [comment]
@catherinepope is correct

**Comment by devinmarieb** [comment]
Thanks @catherinepope !

File: jekyll/_cci2/runner-installation-linux.adoc [hunk 8]
```diff
@@ -10,22 +10,45 @@ version:
...
 {% include snippets/runner-platform-prerequisites.adoc %}
 
+toc::[]
+
+[#download-the-launch-agent-script]
+== Download the launch agent script
+
+Save the link:https://raw.githubusercontent.com/CircleCI-Public/runner-installation-files/main/download-launch-agent.sh[launch-agent.sh] script file to an easily accessible location. From that location, install the launch agent binary for your target platform, either x86_64, or ARM64:
+
+```script
```
Comment (kbatuigas) [inline jekyll/_cci2/runner-installation-linux.adoc:24]: Not actually 100% sure myself: do we use `script` or `shell` for these?

File: jekyll/_cci2/runner-installation-mac.adoc [hunk 9]
```diff
@@ -16,10 +16,25 @@ This page describes how to install CircleCI self-hosted runner on macOS.
 
 toc::[]
 
+[#download-the-launch-agent-script]
+== Download the launch agent script
+
+Save the link:https://raw.githubusercontent.com/CircleCI-Public/runner-installation-files/main/download-launch-agent.sh[launch-agent.sh] script file to an easily accessible location. From that location, install the launch agent binary for your target platform, either x86_64, or M1 (Apple silicon):
+
+```script
```
Comment (kbatuigas) [inline jekyll/_cci2/runner-installation-mac.adoc:24]: Same comment as before

**Pull request merged**